src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -323,6 +324,9 @@ public function channel(): \AMQPChannel
                 throw new \AMQPException(sprintf('Could not connect to the AMQP server. Please verify the provided DSN. (%s)', json_encode($credentials)), 0, $e);
             }
             $this->amqpChannel = $this->amqpFactory->createChannel($connection);
+            if (isset($this->connectionConfiguration['prefetch_count'])) {
```

Comment:
```suggestion

            if (isset($this->connectionConfiguration['prefetch_count'])) {
```

Reply:
I don't understand. What is the difference ?

Reply:
a blank line before the "if"?